app/presenters/transaction_presenter.rb

```diff
@@ -48,6 +50,12 @@ def start_button_text
     end
   end
 
+  def show_country_notice?
+    return false if Date.today < (Date.new(2020,04,01) + 14.days)
```

Comment:
Eeep wrong year 🤦

Reply:
epppp... no wonder it was being funny ......

Reply:
good spot, thank you